lib/Sema/CSFix.h

```diff
@@ -103,8 +103,13 @@ enum class FixKind : uint8_t {
   /// Add explicit `()` at the end of function or member to call it.
   InsertCall,
 
-  /// Add one or more property unwrap operators ('$')
-  InsertPropertyWrapperUnwrap,
+  /// Add a '$' to refer to the property wrapper type instead of the
+  /// wrapped property type.
+  UsePropertyWrapperType,
```

Comment:
Nit: I think we can drop `Type` form here since really you'd want to use property decl and subsequently its type.